src/categorical_algebra/CSets.jl

```diff
@@ -774,16 +824,25 @@ function backtracking_search(f, X::ACSet, Y::ACSet;
   if iso isa Bool
     iso = iso ? Ob : ()
   end
-  for c in iso
-    nparts(X,c) == nparts(Y,c) || return false
-  end
   if monic isa Bool
     monic = monic ? Ob : ()
   end
-  # Injections between finite sets are bijections, so reduce to that case.
+  isoFailures = Iterators.filter(c->nparts(X,c)!=nparts(Y,c),iso)
+  monoFailures = Iterators.filter(c->nparts(X,c)>nparts(Y,c),monic)  
```

Comment:
Our convention is to use snake case for variable names, so these should be `iso_failures` and `mono_failures`.